README.md

```diff
@@ -221,11 +217,22 @@ pancake --verbose
 
 💡 You can use Pancake with your own modules. All you have to do in your modules is:
 
-1. add the Pancake module object to your `package.json` file
-2. add the Pancake `postinstall` script and dependency to your `package.json` file
-3. and add your peer dependencies
+1. Install Pancake
+2. Add the Pancake module object to your `package.json` file
+3. Add the Pancake `postinstall` script and dependency to your `package.json` file
+4. Add your peer dependencies
+
+
+### 1. Install Pancake
+
+To install pancake use node package manager.
+
+```
+npm i @gov.au/pancake
```

Comment:
add a `$`  before  `npm i @gov.au/pancake` ?

Reply:
We discussed this, decided not to add the `$`